src/components/Timeline/index.js

```diff
@@ -96,19 +75,35 @@ export const Timeline = ({
     rootAudioElm.currentTime = time / 1000
     setTimeCursorTime(time)
   })
-
+  const Container = styled("div")(() => ({
+    width,
+    overflow: "hidden",
+    position: "relative",
+    height: 64,
+    cursor: "pointer",
+    borderBottom: `1px solid ${themeColors.Selection}`,
+    color: themeColors.fg,
+  }))
+  const TimeCursor = styled("div")(() => ({
+    position: "absolute",
+    width: 0,
+    height: 0,
+    top: 0,
+    left: ((timeCursorTime - visibleTimeStart) / visibleDuration) * width - 6,
+    borderLeft: "8px solid transparent",
+    borderRight: "8px solid transparent",
+    borderTop: `12px solid ${themeColors.green}`,
+  }))
   return (
     <Container
       ref={containerRef}
-      themeColors={themeColors}
-      width={width}
       onClick={rootAudioElm ? onClickTimeline : undefined}
     >
       {range(timeTextCount).map((timeTextIndex) => (
         <TimeText
           key={timeTextIndex}
           x={(timeTextIndex / timeTextCount) * width}
-          faded={timeTextTimes[timeTextIndex] < 0}
+          faded={(timeTextTimes[timeTextIndex] < 0).toString()}
```

Comment:
this should be a boolean not a string, `.toString()` will make this always truthy now